libs/CCAppCommon/src/ccDisplayOptionsDlg.cpp

```diff
@@ -91,26 +93,33 @@ ccDisplayOptionsDlg::ccDisplayOptionsDlg(QWidget* parent)
 
 	// fill the application style combo-box
 	{
-		// store the default style (= the active one when the dialog is first shown)
-		QStyle* defaultStyle = ccApp->style();
-		QString defaultAppStyle;
-		if (defaultStyle)
+		// Store the default style (= the active one when the dialog is first shown)
+		// Which is necessarily the key currently stored in persistent settings
+		// since this is saved each time the current setting is selected,
+		// and since it's the first time the user opens the settings for this session,
+		// then, the style was not changed.
+		QSettings settings;
+		settings.beginGroup(ccPS::AppStyle());
+		const QString defaultStyleName = settings.value("style").toString();
+		settings.endGroup();
+
+				// fill the combo-box
```

Comment:
strange formatting or is it Github visualization?

Reply:
strange formatting